cmd/kube-proxy/app/validation.go

```diff
@@ -87,6 +87,30 @@ func validateKubeProxyIPTablesConfiguration(config componentconfig.KubeProxyIPTa
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("MinSyncPeriod"), config.MinSyncPeriod, "must be greater than or equal to 0"))
 	}
 
+	if config.MinSyncPeriod.Duration > config.SyncPeriod.Duration {
```

Comment:
why checking this both here and L108?

Reply:
That's an interesting finding :)

Because both iptables and ipvs proxy has the `SyncPeriod` and `MinSyncPeriod` - they don't share the same `SyncPeriod` and `MinSyncPeriod`.

Reply:
We need to validate them both in iptables and ipvs mode.

Reply:
Cool! I didn't notice this :)